Gopkg.lock

```diff
@@ -49,17 +49,17 @@
 [[projects]]
   name = "github.com/fabric8-services/fabric8-notification"
   packages = ["design"]
-  revision = "master"
+  revision = "907d3253a9e8ae42f61ce7c931fd920df20ee0ec"
 
 [[projects]]
   name = "github.com/fabric8-services/fabric8-tenant"
   packages = ["design"]
-  revision = "master"
+  revision = "2614bf1824754901b7e2f9d7b4ad4d7e3bac61ff"
 
 [[projects]]
   name = "github.com/fabric8-services/fabric8-wit"
   packages = ["design"]
-  revision = "master"
+  revision = "b59d0dc8ae9b2bfea1fb5dc6ff3552c42b735756"
```

Comment:
Goodness, we were using master all along? :) 
I think this happened during the migration from `glide` to `dep` because I'm pretty sure we were using a commit has prior to this.

Reply:
Yes. But it wasn't critical. We control these repos and don't use anything besides generated clients.
But worth fixing anyway :)